test/test.js

```diff
@@ -869,88 +847,12 @@ describe('CodeService', function () {
       }
     });
 
```

Comment:
The two tests below that are removed and the test above ('should error if no API Key is supplied') made we wonder if there should be a test for when an invalid API key is provided. Since the API key functionality was added at a different time, maybe there is a test/reason I missed that covers this/why this isn't needed.

Reply:
I added a test for invalid API Key in [e0f6fda](https://github.com/cqframework/cql-exec-vsac/pull/27/commits/e0f6fdaee26cd4e0456388962c376da585bee218).